### PR TITLE
Github actions 

### LINKED ISSUE #33: Set up Github Actions
Github actions need to be set up

Why is it needed?
- If `npm run build` fails for a particular PR/commit, we will get notified that the build is failing for this one. So that we can avoid merging it.
- If such PR gets merged, the build will fail on production which is not desirable.
- Knowing problems beforehand will help us fix them earlier.

What to do?
- run `npm ci` (why not `npm install`? refer this one: https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-nodejs)
- run `npm run build` 
- TODO: run for tests once we have them in place

Refer: https://github.com/Real-Dev-Squad/website-crypto/blob/develop/.github/workflows/node.js.yml

### PR DESCRIPTION
**Pull request by agarwalamn**
# Closes #33 
Added node.js.yml file for issue #34 

As we are using yarn to build things used yarn commands

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: yarn
```
Comment (swarajpure) [inline .github/workflows/node.js.yml:27]: Not `yarn install`?

**Review by swarajpure — COMMENTED** [review]
Thank you!

I have a doubt. Please clarify.

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (agarwalamn) [reply .github/workflows/node.js.yml:27]: yarn and yarn install works the same

Comment (swarajpure) [reply .github/workflows/node.js.yml:27]: Cool.

Also, can the name of file be self-explanatory?

Something like `continuous-integration`?

Comment (agarwalamn) [reply .github/workflows/node.js.yml:27]: sure, I thought we have node.js as the default schema for the file name😅

**Comment by agarwalamn** [comment]
@swarajpure renamed the file

**Review by swarajpure — APPROVED** [review]
Thank you! 😄

**Pull request merged**